scripts/deploy.js

```diff
@@ -100,9 +100,9 @@ async function deployAirdrop(semaphoreAddress) {
             ])
         ),
     })
-    spinner.text = `Waiting for Semaphore deploy transaction (tx: ${tx.hash})`
+    spinner.text = `Waiting for SemaphoreAirdrop deploy transaction (tx: ${tx.hash})`
```

Comment:
@recmo if the name/naming consistency is important I would move it to const. Found a few other occurrences of the `Semaphore` above in the code.